pkg/server/controllers.go

```diff
@@ -613,23 +661,54 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 		}
 
 		go c.Start(goContext(hookContext), 2)
-		go permissionClaimLabelController.Start(goContext(hookContext), 5)
-		go permissionClaimLabelResourceController.Start(goContext(hookContext), 2)
 
 		return nil
 	}); err != nil {
 		return err
 	}
 
+	server.AddPostStartHook(postStartHookName(permissionclaimlabel.ControllerName), func(hookContext genericapiserver.PostStartHookContext) error {
```

Comment:
Let's reorder to be consistent so it's

1. Setup stuff for controller X
2. Poststart hook for controller X

(repeat for n controllers)